chain/network-primitives/src/network_protocol/edge.rs

```diff
@@ -4,6 +4,13 @@ use near_primitives::borsh::maybestd::sync::Arc;
 use near_primitives::hash::CryptoHash;
 use near_primitives::network::PeerId;
 
+use crate::time::Utc;
```

Comment:
import just crate::time;
time::Utc is concise enough.

Reply:
done.